multigeiger/display.cpp

```diff
@@ -106,28 +106,33 @@ void DisplayGMC(int TimeSec, int RadNSvph, int CPS, bool use_display, bool conne
   if (!isLoraBoard) {
     char output[80];
     int TimeMin = TimeSec / 60;         // calculate number of minutes
-    if (TimeMin >= 999) TimeMin = 999;  // limit minutes to max. 999
+    TimeMin = TimeMin % 999;  // limit minutes to max. 999, roll over
 
     // print the upper line including time and measured radation
+    pu8x8->setFont(u8x8_font_open_iconic_embedded_1x1);
+    sprintf(output, "%c", 0x4A);
+    if (ble_active) pu8x8->print(output);
     pu8x8->setFont(u8x8_font_7x14_1x2_f);
 
-    if (TimeMin >= 1) {                 // >= 1 minute -> display in minutes
-      sprintf(output, "%3d", TimeMin);
+    if (TimeSec < 60)    {    // < 1 minute -> display in seconds
+      sprintf(output, "%2ds", TimeSec);
       pu8x8->print(output);
-    } else {                            // < 1 minute -> display in seconds, inverse
-      sprintf(output, "%3d", TimeSec);
-      pu8x8->inverse();
+    } else {                  // >= 1 minute -> display in minutes
+      sprintf(output, "%3d", TimeMin);
       pu8x8->print(output);
-      pu8x8->noInverse();
     }
-    sprintf(output, "%7d nSv/h", RadNSvph);
+    if (ble_active) {
+      sprintf(output, "%6d nSv/h", RadNSvph);
+    } else {
+      sprintf(output, "%7d nSv/h", RadNSvph);
+    }
```

Comment:
see "nothing shifts around" comment above.

Reply:
also, a good rule is to always remember the `cond ? if_true : if_false` way to write a term when one wants to write an otherwise duplicate if/else block.

Reply:
I was not so sure whether that works with two full commands... will adapt.

Reply:
Actually, reverted it to single %6d line to avoid shifting text.